### ISSUE TITLE
New tasks are not automatically showing or no way to refresh

### ISSUE THREAD

**Issue by anbuselvan** [sana-ajani/taskrunner-code#31]
I'm only using the default **tasks.json** inside **.vscode** folder to clarify this issue.

When I add new task it doesn't show up automatically in the sidebar.

<img width="500" alt="Screen Shot 2021-08-04 at 5 24 34 PM" src="https://user-images.githubusercontent.com/167158/128272656-4a87f24b-293a-4c51-8bf8-e2d5616eb2b3.png">

The new tasks are showing only after relaunching vscode.

<img width="500" alt="Screen Shot 2021-08-04 at 5 25 05 PM" src="https://user-images.githubusercontent.com/167158/128272698-acfe2389-3dd4-4baa-8713-28b5404e6a0b.png">

**Comment by MatthewEskolin** [comment]
extension.ts registers a command called taskOutline.refresh

Refresh Icon should be visible in the Task Runner view

![image](https://user-images.githubusercontent.com/8338858/133322794-f7ecb1d5-f066-4e17-b734-b536ff12e3d0.png)


I am also able to run this command and see the Task List refreshed by creating a keyboard shortcut for this command.

![image](https://user-images.githubusercontent.com/8338858/133320617-6e11cb73-3a38-49e4-acd9-2d1677515609.png)

**Comment by willynilly** [comment]
Definitely needs to be fixed.  Also, sometimes when I initially load a project, it does not show my tasks and I have to Reload the Window.

**Comment by Wolfy7** [comment]
On my Windows PC it shows the refresh icon:
![grafik](https://user-images.githubusercontent.com/16044116/154631697-ddf98093-9500-4112-ad43-2a1cfa87097b.png)
Version: 1.64.2 (user setup)
Commit: f80445acd5a3dadef24aa209168452a3d97cc326
Date: 2022-02-09T22:02:28.252Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19044

But on an Linux PC (connected via xrdp) not:
![grafik](https://user-images.githubusercontent.com/16044116/154631833-c6e3b4ba-adfe-4946-bf0b-518a2a9a82a4.png)
Version: 1.64.2
Commit: f80445acd5a3dadef24aa209168452a3d97cc326
Date: 2022-02-09T22:02:29.527Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.13.0-28-generic snap
